config.json

```diff
@@ -944,6 +944,18 @@
         "conditionals"
       ]
     },
+    {
+      "uuid": "1d21cd68-10ac-427d-be6d-77152bceacc4",
+      "slug": "rational-numbers",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 2,
+      "topics": [
+        "classes",
+        "arithmetic",
+        "operator_overloading"
```

Comment:
`operator_overloading` is not on the list of valid topics. Please refer to the [global topics list](https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt) for valid topics.